module/Order/src/Order/Entity/Traits/ArraySerializableTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Order\Entity\Traits;
+
+
+trait ArraySerializableTrait
```

Comment:
if this trait can be shared with other entities then we should not place it inside order/* directory.